designs/index.md

```diff
@@ -14,27 +14,37 @@ changes as designs are implemented.
 Always go to the [documentation]({{ site.docs_site_url }}) for descriptions of
 current Bazel functionality.
 
-## Table of Contents
-{:.no_toc}
-
-* ToC
-{:toc}
-
-
 ## When do I need a design document and review?
```

Comment:
The headers phrased as questions makes the TOC harder to scan. Perhaps update the headers to be more concise?
Additionally, there seem to be two audiences/ sets of instructions in this page: authors and reviewers. Perhaps make it easier to scan for that.

Reply:
Removed question headers